### ISSUE TITLE
md-icon-button:  ripple just effect once

### ISSUE THREAD

**Issue by HRan2004** [material-components/material-web#4088]
```
<md-standard-icon-button class={Style.menu}>
  <span className="material-icons-outlined">menu</span>
</md-standard-icon-button>
```
I use icon-button in react, the ripple motion only show on i first click it

**Comment by kkachniarz220** [comment]
+1

**Comment by tanishqmanuja** [comment]
I can confirm the same for angular on devices like mobile and tablets. On Chrome desktop it works fine until you turn on device mode (ctrl+shift+M)

**Comment by tanishqmanuja** [comment]
Works now on release `v1.0.0-pre.8` as expected.

![brave_6C9bWmen6h](https://github.com/material-components/material-web/assets/40914272/9e20ba1c-3459-4cca-a661-d3b9ef4502c0)

**Comment by asyncLiz** [comment]
@HRan2004 can you verify if this is fixed in react for v1.0.0-pre.8?
